hello_world.py

```diff
@@ -1 +1,2 @@
-print("Hello world")
\ No newline at end of file
+print("Hello world")
+print("Hi from Python")
```

Comment:
Add a new line at the end of the script.

Reply:
Resolved